app/lib/primer/classify.rb

```diff
@@ -3,11 +3,20 @@
 module Primer
   # :nodoc:
   class Classify
+    # Load the utilities.yml file.
+    # Disabling because we want to load symbols, strings, and integers from the .yml file
+    # rubocop:disable Security/YAMLLoad
+    UTILITIES = YAML.load(
+      File.read(
+        File.join(File.dirname(__FILE__), "./classify/utilities.yml")
+      )
+    ).freeze
+    # rubocop:enable Security/YAMLLoad
+
```

Comment:
Does this seem like the best place to load the file?

Reply:
🤷🏻 works for me!